modules/installation-creating-azure-dns.adoc

```diff
@@ -19,6 +19,13 @@ You must configure networking and load balancing in Microsoft {cp} for your
 {product-title} cluster to use. One way to create these components is
 to modify the provided Azure Resource Manager (ARM) template.
 
+ifdef::ash[]
+Load balancing requires the following DNS records:
+
+* An `api` DNS record in the public zone for the API public load balancer.
+* An `api-int` DNS record in the private zone for the API internal load balancer.
+endif::ash[]
```

Comment:
```suggestion
* An `api` DNS record for the API public load balancer in the DNS zone .
* An `api-int` DNS record for the API internal load balancer in the DNS zone.
endif::ash[]
```
The main change I am recommending here is to remove "private DNS zone." Azure Stack does not have "private" zones. It does not actually have "public" zones either, but just "DNS zones."